demo/js/MIDI/LoadPlugin.js

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
3rd party code without reference? Either use a git submodule or contribute to the project to release a npm package